[Term entry] Kotlin math_method: acosh() (#3799)
Codecademy/docs: content/kotlin/concepts/math-methods/terms/acosh/acosh.md

---
Title: 'acosh()'

> Description: 'Returns the hyperbolic arccosine of a numeric argument.'

Reviewer: ```suggestion
Description: 'Returns the inverse hyperbolic cosine of a numeric argument.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Kotlin'
  - 'Math'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

The **`acosh()`** function returns the hyperbolic arccosine of a number.

## Syntax

```pseudo
acosh(x)
```

> The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` datatype.
> 
> It returns the hyperbolic arccosine of the argument, which represents a real number.

Reviewer: ```suggestion
The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` and `Float` datatype. 

> **Note**: It returns the hyperbolic arccosine of the argument, which represents a real number.
```
We can take float datatype also. [ref](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/acosh.html)

## Example

The example below demonstrates how to use `acosh()` to calculate the hyperbolic arccosine of `2.0`.

```kotlin

> import kotlin.math.acosh
> fun main() {
>     // Calculate the hyperbolic arccosine of 2.0
>     val x = 2.0
>     val result = acosh(x)
> 
>     // Print the result
>     println(result)

Reviewer: ```suggestion
import kotlin.math.acosh
fun main() {
    val x = 2.0
    val result = acosh(x)

    println(result)
```
No need for code comments here
